docs/quickstart.md

```diff
@@ -1,75 +1,46 @@
 # Quickstart
 
-## Installation Script(Deprecated)
+## On macOS
 
-!!! note
-      The installation script method is deprecated as of version 0.7.
-
-=== "Linux"
+- Only supported: Apple Silicon (M series), macOS 14+
+- [Download the installer](https://gpustack.ai)
 
-    GPUStack provides a script to install it as a systemd service on Linux with default port 80. To install GPUStack using this method, just run:
-
-    ```bash
-    curl -sfL https://get.gpustack.ai | sh -s -
-    ```
+## On Windows
 
-=== "macOS"
+- Only supported: win 10, win 11
+- [Download the installer](https://gpustack.ai)
 
-    GPUStack provides a script to install it as a launchd service on macOS with default port 80. To install GPUStack using this method, just run:
+## On Linux
 
-    ```bash
-    curl -sfL https://get.gpustack.ai | sh -s -
-    ```
+We provide two installation methods on Linux:
 
-=== "Windows"
+- **Docker installation**(Recommended)
+- **pip installation**
 
-    Run PowerShell as administrator (**avoid** using PowerShell ISE), then run the following command to install GPUStack with default port 80:
-
-    ```powershell
-    Invoke-Expression (Invoke-WebRequest -Uri "https://get.gpustack.ai" -UseBasicParsing).Content
-    ```
-
-## Desktop Installer
-
-=== "macOS"
-
-    - Only supported: Apple Silicon (M series), macOS 14+
-    - [Download the installer](https://gpustack.ai)
-
-=== "Windows"
+### Docker Installation (Recommended)
```

Comment:
Provide the command for CUDA to help users get started. For other devices, we don't need to list everything here. Guide users to https://docs.gpustack.ai.